packages/core/admin/admin/src/pages/Home/components/ContentManagerWidgets.tsx

```diff
@@ -49,84 +41,98 @@ const DocumentStatus = ({ status = 'draft' }: DocumentStatusProps) => {
   );
 };
 
-const WidgetContent = ({ document }: { document: RecentDocument }) => {
+const RecentDocumentsTable = ({ documents }: { documents: RecentDocument[] }) => {
   const { formatMessage } = useIntl();
   const { trackUsage } = useTracking();
   const navigate = useNavigate();
 
+  const getEditViewLink = (document: RecentDocument): string => {
+    // TODO: import the constants for this once the code is moved to the CM package
+    const kindPath = document.kind === 'singleType' ? 'single-types' : 'collection-types';
+    const queryParams = document.locale ? `?plugins[i18n][locale]=${document.locale}` : '';
+
+    return `/content-manager/${kindPath}/${document.contentTypeUid}/${document.documentId}${queryParams}`;
```

Comment:
should the documentId be in there for single types?

Reply:
you're right! it somehow didn't 404 🤷